src/posts/post.swagger.ts

```diff
@@ -0,0 +1,197 @@
+import { applyDecorators, Type } from '@nestjs/common';
+import {
+  ApiOperation,
+  ApiResponse,
+  ApiBearerAuth,
+  getSchemaPath,
+  ApiExtraModels,
+} from '@nestjs/swagger';
+import { ResPostDto } from './dto/res-post.dto';
+import { ApiResponseDto } from './dto/api-response.dto';
+
+/**
+ * 인증 실패 공통 응답
+ */
```

Comment:
jsdoc이라는 문법이 있는데 아래처럼 표현하면 IDE에서 마우스 호버시에 설명이 보이는거로 알고있어요
물론 현재도 괜찮고 주석도 컨벤션을 팀원들끼리 정할 수 있어서 한번 얘기해보는 정도도 추천드립니다

```suggestion
/**
 * @description 인증 실패 공통 응답
 */
```